lib/camunda-cloud/NavigatedViewer.js

```diff
@@ -0,0 +1,39 @@
+import inherits from 'inherits';
+
+import BaseViewer from '../base/NavigatedViewer';
+
+import iconRendererModule from '@bpmn-io/element-templates-icons-renderer';
+
+import drilldownModule from './features/drilldown';
+
+import zeebeModdle from 'zeebe-bpmn-moddle/resources/zeebe.json';
+
+
+/**
+ *
+ * @param {Object} options
+ */
+export default function NavigatedViewer(options = {}) {
```

Comment:
Every time we add a module to the `Viewer`, we need to copy the same change to the `NavigatedViewer`. Is there maybe any way to make this easier? Probably not because there is no double inheritance...

Reply:
We could add a `commonViewerModules` utility, that get's included in both viewer and navigatedViewer. I'll try it out and see how it feels